src/actions/CreateWalletActions.tsx

```diff
@@ -42,7 +52,7 @@ export const createWallet = (account: EdgeAccount, opts: EdgeCreateCurrencyWalle
 
     // Update most recent wallets list
     if (out != null) {
-    const walletId = out.id
+      const walletId = out.id
```

Comment:
Apply to previous commit (per previous feedback comment)

Reply:
I made changes after you started reviewing it seems, no longer see this